### DISCUSSION TITLE
Convert PDF to image

### DISCUSSION THREAD

**Discussion by bennycode** [Hopding/pdf-lib#1666] [General]
Is it possible to convert a PDF into a PNG image using pdf-lib?

**Comment by Gowthamraj-tai** [comment]
bro did you done this conversion

**Reply by bennycode** [reply]
I followed this tutorial: https://typescript.tv/hands-on/analyzing-pdfs-with-chatgpt-using-openais-vision-api/#convert-pdfs-to-images-with-nodejs
